README.md

```diff
@@ -135,22 +370,114 @@ batch(() => {
 console.log('Process end');
 ```
 
-With the following output:
+output:
 
 ```text
 Arsène Lupin
 Sherlock Holmes
 Process end
 ```
+> [!NOTE]
+> - Managing stores in a batch context can be quite challenging (getting store values inside a batch, subscribing in a batch, avoiding rerunning calculations outside the batch if the value has been updated inside, ...). All potential situations have been specified in the specs.
+> - `batch` can be called inside `batch`. The subscriber calls are performed at the end of the first batch, synchronously.
 
-## Installation
 
-You can add Tansu to your project by installing the `@amadeus-it-group/tansu` package using your favorite package manager, ex.:
+### asReadable
 
-* `yarn add @amadeus-it-group/tansu`
-* `npm install @amadeus-it-group/tansu`
+[api documentation](https://amadeusitgroup.github.io/tansu/functions/asReadable.html)
 
-## Usage
+`asReadable` transforms a writable store into a readable one. A readable store is a store without the set and update method.
```

Comment:
Note that it is not mandatory for the store passed to `asReadable` to actually be writable.